examples/rapids/mlflow_project/README.md

```diff
@@ -0,0 +1,47 @@
+### Train and Publish Locally With RAPIDS and MLflow
+**[RAPIDS](https://rapids.ai/)** is a suite of open source libraries for GPU-accelerated analytics. 
+
+**[RAPIDS cuML](https://github.com/rapidsai/cuml)** matches the scikit-learn API, so it can build on MLflow's existing support for scikit-learn-like models to support
+persistence and deployment."
+
+The example workflows below train RAPIDs regression models to predict airline flight delays, using MLflow to log models and deploy them as local REST API endpoints for real-time inference.
+
+#### Jupyter Notebook Workflow
+[Jupyter Notebook](notebooks/rapids_mlflow.ipynb)
+
+#### CLI Based Workflow
+1. Download data
+    1. `cd examples/rapids/mlflow_project`
+        ```shell script
+        # Download the file
+        wget -N https://rapidsai-cloud-ml-sample-data.s3-us-west-2.amazonaws.com/airline_small.parquet
+        ```
```

Comment:
We should include a test for the CLI-based workflow - it should be doable by adding an item to the list of `(directory, args_to_mlflow_run)` tuples in the existing [examples tests](https://github.com/mlflow/mlflow/blob/91dca77a7e850ec25a88dd79736cbab2bfd740a3/tests/examples/test_examples.py#L59). For example, you should be able to add an item like `(os.path.join("rapids", "mlflow_project"), ["-e", "simple"])` and the test will invoke `mlflow run rapids/mlflow_projects -e simple` and verify that it succeeds.

It might be easiest to test this if we update the project so that `fpath` is actually an optional parameter and when unspecified, the training script downloads the data via logic like:

```
import requests
import tempfile

# Data will ultimately be written to the file whose path is stored in fname
_, fname = tempfile.mkstemp()
url = "https://rapidsai-cloud-ml-sample-data.s3-us-west-2.amazonaws.com/airline_small.parquet"
airline_data_response = requests.get(url)
with open(fname , 'wb') as handle:
    handle.write(airline_data_response.content)
```

We can defer this to a follow-up PR, but would be good to have - otherwise, this example may break in the future.

Reply:
Ah, just discussed offline with @harupy - looks like our underlying CI provider (Github actions) doesn't support GPUs, so we unfortunately can't actually test this example - feel free to disregard the above comment.